html/pfappserver/lib/pfappserver/Model/DB.pm

```diff
@@ -157,6 +157,18 @@ sub create {
 
 =head2 secureInstallation
 
+Intented to integrate the "/usr/bin/mysql_secure_installation" steps
+
+1. Set a password for the database “root” user (different from the Linux root user!), which is blank by default;
+
+2. Delete “anonymous” users, i.e. users with the empty string as user name;
+
+3. Ensure the root user can not log in remotely;
+
+4. Remove the database named “test”;
```

Comment:
What are these double quotes... These don't look like normal double quotes.

They are almost the same like Bobbee and Zayme but they aren't

Imagine if Bobbee started rebooting random servers instead of Zayme and Zayme started picking up a ton of nerf darts on the ground while cleaning our office.

I would call this the beginning of the apocalypse.

Is this really what we want ???

So I would suggest you remove your Bobbee and replace it with good old Zayme